test/MultiVesting.ts

```diff
@@ -152,4 +182,39 @@ describe("MultiVesting", function () {
     console.log("can update vest when cliff less than older and amount = 0");
     await vesting.vest(await receiver4.getAddress(), await currentTimestamp(), 1000, 0, 50)
   })
+
+  it("Vesting created for passed timestamp (duration >= cliff)", async () => {
+    let day = 60 * 60 & 24
+    let sixtyDays = day * 60
+    let currentTime = await currentTimestamp()
+    let oldTimestamp = currentTime - sixtyDays
+
+    await cheel.connect(gnosisCheel).mint(vesting.address, 1000)
```

Comment:
`1000` - let's make it as a named constant. Potentially replace it with `faker.number`